i18n/es.json

```diff
@@ -37,7 +37,9 @@
   "I18N_FOOTER_BROWSE_LIBRARY": "Navega en la librería",
   "I18N_FOOTER_CONTACT_ALL_CAPS": "CONTÁCTANOS",
```

Comment:
Ah, I think you forgot to remove this line. That's why the backend test is failing.